stable/nexus/templates/pvc.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  annotations:
+  {{- if .Values.persistence.storageClass }}
```

Comment:
Can we move to the new pattern for storageClass. See https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/alertmanager-pvc.yaml#L20 and https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L104-L110